modules/pvc-autoscale/data/pvc_autoscale_notebook.json

```diff
@@ -0,0 +1,162 @@
+{
+  "cells": [
+    {
+      "content": "# Setup\nInstall required packages and configure pods for testing.",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "## Setup Kubernetes\nSetup kubernetes objects for the test.",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pods | app =~ \"shoreline\" | limit=1 | `kubectl apply -f  \"/tmp/pvc_test.yaml\"`",
```

Comment:
When you run `make test`, it creates `pvc_test.yaml` from `pvc_test.yaml.template`.
That was because it used to substitute a cluster variable.
But now, that means that if you don't run `make test` first, doing `make install` or `terraform apply` fails.
So, I think you can just copy the template to the flat yaml now and check it in.
Maybe also add a note that it can take some time for the file objects to propagate.